packages/treasury/test/test-interest.js

```diff
@@ -49,11 +49,11 @@ test('basic charge 1 period', async t => {
     latestInterestUpdate: 0n,
     interest: AmountMath.makeEmpty(brand),
   };
-  // 6n is daily interest of 2.5% APR on 100k. Compounding is in the noise.
+  // 7n is daily interest of 2.5% APR on 100k. Compounding is in the noise.
   t.deepEqual(calculator.calculate(debtStatus, ONE_DAY), {
     latestInterestUpdate: ONE_DAY,
-    interest: AmountMath.make(brand, 6n),
-    newDebt: AmountMath.make(brand, 100006n),
+    interest: AmountMath.make(brand, 7n),
+    newDebt: AmountMath.make(brand, 100007n),
```

Comment:
mixing this rounding change in with this PR is hurting my brain a little. It feels like a very different angle to review from...